pkg/tdex-market-loader/client.go

```diff
@@ -0,0 +1,321 @@
+package tdexmarketloader
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"github.com/tdex-network/tdex-protobuf/generated/go/trade"
+	"github.com/tdex-network/tdex-protobuf/generated/go/types"
+	"golang.org/x/net/proxy"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strings"
+)
+
+const (
+	registryUrl   = "https://api.github.com/repos/tdex-network/tdex-registry/contents/registry.json"
```

Comment:
if we want to support testnet as well, just in case, we have in the branch testnet a different list of providers

https://github.com/tdex-network/tdex-registry/tree/testnet

Reply:
Done